### DISCUSSION TITLE
How do we refresh Forge?

### DISCUSSION THREAD

**Discussion by pbiggar** [kahole/edamagit#168] [Q&A]
I'm using Edamagit in VSCode, and have Forge enabled. Pressing `g` doesn't refresh the pull requests for me. In magit, `' ' f` would refresh - is there a way to do it in Edamagit?

**Comment by kahole** [comment]
Currently it is cached and refreshed based on some timers and logic.
So it eill refresh, but maybe not right away.
I will look into having a "force forge refresh" command like the one you suggested from magit. 👍🏻

**Reply by pbiggar** [reply]
Thanks! What are the timers set to? Looks like it's not refreshing for me. It looks like it did eventually update, but it took a long time.
